apps/science/ScienceConfig.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "../targets/science/PinNames.h"
```

Comment:
hmm using a relative path here is a little odd. i believe PinNames.h is included in the path in the makefile. try without specifying the whole path.